src/App.tsx

```diff
@@ -26,25 +26,58 @@ function App() {
     const yearlyHeaders = [];
 
     for (let i = 0; i < 30; i++) {
-      yearlyHeaders.push({ name: 'Y' + (new Date().getFullYear() + i)});
+      yearlyHeaders.push({ name: "Y" + (new Date().getFullYear() + i) });
     }
 
+    // This part increases the width of a chosen column
+    const nameCol = worksheet.getColumn('B');
+    nameCol.width = 38;
```

Comment:
is this used anywhere?

Reply:
That line is used to make the column B in the excel sheet bigger. Thought it would make it look nicer

Reply:
ah, that makes sense.  And I think overall it looks very nice, feel free to merge when you are happy with it 🚢